pandas/core/generic.py

```diff
@@ -2042,6 +1932,114 @@ def to_excel(
         verbose=True,
         freeze_panes=None,
     ) -> None:
+        """
+            Write {klass} to an Excel sheet.
```

Comment:
Too much indentation, you need to dedent the whole docstring four spaces except the opening quotes.

Reply:
done in the latest commit